app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_comments_counter
+    post.increment!(:like_counter)
+  end
```

Comment:
- [OPTIONAL] I think this should be `update_likes_counter`, please kindly fix the typing error.